modules/server-node/prisma-sqlite/schema.prisma

```diff
@@ -184,6 +183,26 @@ model EventSubscription {
 }
 
 model OnchainTransaction {
+  // Response fields
+  id String @id @default(uuid())
+
+  // Channel fields
+  status         String  @default("submitted") // no enums
+  reason         String // no enums
+  error          String?
+  channelAddress String
+  channel        Channel @relation(fields: [channelAddress], references: [channelAddress])
+
+  createdAt DateTime @default(now())
```

Comment:
So, noticed that this table has `createdAt`, OnchainTransactionAttempt has `createdAt`, and OnchainTransactionReceipt has `timestamp` (which is a string). Could we maybe just use one format for all of these? Either a timestamp string type or createdAt Date type?